recipes-app/node-red-gpio/node-red-gpio_0.0.6-IOT2050.bb

```diff
@@ -17,12 +17,9 @@ MAINTAINER = "chao.zeng@siemens.com"
 SRC_URI = " \
     git://github.com/node-red/node-red-nodes;protocol=https \
     file://0001-add-the-board-info-add-the-led-control-node.patch \
```

Comment:
Why was that not submitted to upstream?

Reply:
I've submitted only related to gpio din - pwm, led and some fixes I'll put in different pr after mraa gets resolved to or I may put it in node-red-nodes on next week, all this about just to make everything in sync and step by step.